crates/proof-of-sql/src/base/commitment/column_bounds.rs

```diff
@@ -228,6 +228,7 @@ impl ColumnBounds {
             | CommittableColumn::Decimal75(_, _, _)
             | CommittableColumn::Scalar(_)
             | CommittableColumn::VarChar(_) => ColumnBounds::NoOrder,
+            CommittableColumn::RangeCheckWord(_) => ColumnBounds::NoOrder,
```

Comment:
I think this could be in the next PR (or totally separate) since this isn't really used in the two functions that this PR is focused on.